docusaurus/vercel.json

```diff
@@ -0,0 +1,360 @@
+      "statusCode": 301
+    },
+    {
+      "source": "/developer-guides/:path*",
+      "destination": "/platform/developer-guides/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/embedded-setup/:path*",
+      "destination": "/platform/embedded-setup/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/enterprise-setup/:path*",
+      "destination": "/platform/enterprise-setup/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/managing-airbyte/:path*",
+      "destination": "/platform/managing-airbyte/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/operating-airbyte/:path*",
+      "destination": "/platform/operating-airbyte/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/operator-guides/:path*",
+      "destination": "/platform/operator-guides/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/understanding-airbyte/:path*",
+      "destination": "/platform/understanding-airbyte/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/using-airbyte/:path*",
+      "destination": "/platform/using-airbyte/:path*",
+      "statusCode": 301
+    },
+    {
+      "source": "/api-documentation",
+      "destination": "/platform/api-documentation",
+      "statusCode": 301
+    },
+    {
+      "source": "/terraform-documentation",
+      "destination": "/platform/terraform-documentation",
+      "statusCode": 301
+    },
+    {
+      "source": "/operator-guides/configuring-airbyte-db",
+      "destination": "/platform/deploying-airbyte/integrations/database",
+      "statusCode": 301
+    },
+    {
+      "source": "/upgrading-airbyte",
+      "destination": "/platform/operator-guides/upgrading-airbyte",
+      "statusCode": 301
+    },
+    {
+      "source": "/catalog",
+      "destination": "/platform/understanding-airbyte/airbyte-protocol",
+      "statusCode": 301
+    },
+    {
+      "source": "/integrations/sources/appstore-singer",
+      "destination": "/integrations/sources/appstore",
+      "statusCode": 301
+    },
+    {
+      "source": "/connector-development/config-based/",
+      "destination": "/platform/connector-development/config-based/low-code-cdk-overview",
+      "statusCode": 301
+    },
+    {
+      "source": "/connector-development/tutorials/cdk-tutorial-python-http/:path*",
+      "destination": "/platform/connector-development/tutorials/custom-python-connector/getting-started",
+      "statusCode": 301
+    },
+    {
+      "source": "/project-overview/changelog",
+      "destination": "/release-notes/",
+      "statusCode": 301
+    },
+    {
+      "source": "/connector-development/config-based/understanding-the-yaml-file/stream-slicers/",
+      "destination": "/platform/connector-development/config-based/understanding-the-yaml-file/partition-router",
+      "statusCode": 301
+    },
+    {
+      "source": "/cloud/dbt-cloud-integration",
```

Comment:
I've tested a few redirections and getting 404s, for example this one, is redirecting to [https://airbyte-docs-git-11059-versioned-docs-airbyte-growth.vercel.app/platform/cloud/dbt-cloud-integration](https://airbyte-docs-git-11059-versioned-docs-airbyte-growth.vercel.app/platform/cloud/dbt-cloud-integration) instead of [https://airbyte-docs-git-11059-versioned-docs-airbyte-growth.vercel.app/platform/cloud/managing-airbyte-cloud/dbt-cloud-integration](https://airbyte-docs-git-11059-versioned-docs-airbyte-growth.vercel.app/platform/cloud/managing-airbyte-cloud/dbt-cloud-integration)